css/style.css

```diff
@@ -264,6 +273,10 @@ button.button-inline:hover {
 	overflow: hidden;
 	display: flex;
 	min-height: 40px;
+	position: fixed;
```

Comment:
Ah, that was a good idea, but yeah, probably some JS will be needed.
I'm more worried about how it would look though. Fixed titles usually require some extra style to distinguish. from the rest of the DOM